src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/utils/index.ts

```diff
@@ -40,12 +19,12 @@ export const ensResolver = createDecorator({
         if (resolvedAddress) {
           return resolvedAddress
         }
+
+        return contractAddress
       } catch (e) {
         console.error(e.message)
         return contractAddress
       }
-
-      return contractAddress
```

Comment:
This can be outside the Try/Catch, you have to delete the one inside the Catch block